paperoni/db/database.py

```diff
@@ -510,3 +510,28 @@ def has_flag(self, paper, flagname):
             if flag.flag_name == flagname:
                 return True
         return False
+
+    def insert_author(self, author_id, name, quality):
+        print("INSIDE insert_author")
```

Comment:
Might want to remove the debug printing here and in the other methods.

Reply:
Oops ! Fixed in [a62861d](https://github.com/mila-iqia/paperoni/pull/20/commits/a62861d856e64cdfdf54f287f38aace67ae53cf7)